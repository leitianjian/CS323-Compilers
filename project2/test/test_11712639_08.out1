Program (1)
  ExtDefList (1)
    ExtDef (1)
      Specifier (1)
        TYPE: int
      FunDec (1)
        ID: test
        LP
        RP
      CompSt (1)
        LC
        StmtList (2)
          Stmt (2)
            Exp (2)
              ID: func
              LP
              Args (2)
                Exp (2)
                  ID: func1
                  LP
                  Args (2)
                    Exp (2)
                      ID: a
                  RP
              RP
            SEMI
          StmtList (3)
            Stmt (3)
              RETURN
              Exp (3)
                INT: 1
              SEMI
        RC
