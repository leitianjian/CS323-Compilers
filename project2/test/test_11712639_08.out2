Error type 1 at Line 2: a variable is used without definition
Error type 2 at Line 2: func1 function is invoked without deﬁnition
Error type 2 at Line 2: func function is invoked without deﬁnition
