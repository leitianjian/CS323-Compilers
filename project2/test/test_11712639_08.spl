int test(){
    func(func1(a));
    return 1;
}