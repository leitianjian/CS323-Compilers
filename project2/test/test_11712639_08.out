Error type 2 at line 2: func function is invoked without definition
Error type 2 at line 2: func1 function is invoked without definition
Error type 1 at line 2: a variable is used without definition
